chimp fails on windows machines

I'm using chimp v0.27.0 (installed locally)
running command
`node_modules\.bin\chimp`
fails with following error:
```
[chimp] Running...
module.js:327
    throw err;
    ^

Error: Cannot find module 'C'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Array.forEach (native)
    at Object.initializer (C:\Users\boda2\WebstormProjects\marketplace-admin\node_modules\cucumber\lib\cucumber\cli\support_code_loader.js:22:25)
    at Object.Library (C:\Users\boda2\WebstormProjects\marketplace-admin\node_modules\cucumber\lib\cucumber\support_code\library.js:122:25)
    at Object.getSupportCodeLibrary (C:\Users\boda2\WebstormProjects\marketplace-admin\node_modules\cucumber\lib\cucumber\cli\support_code_loader.js:10:58)
    at Object.getSupportCodeLibrary (C:\Users\boda2\WebstormProjects\marketplace-admin\node_modules\cucumber\lib\cucumber\cli\configuration.js:127:32)
    at Object.getSupportCodeLibrary (C:\Users\boda2\WebstormProjects\marketplace-admin\node_modules\cucumber\lib\cucumber\runtime.js:43:46)
Cucumber steps failed
```

Quick research gave me following information:
default chimp options contain compiler value, which is
`'js:' + path.resolve(__dirname, '../lib/babel-register.js')`
from https://github.com/xolvio/chimp/blob/068b89a5fcbac72874214d8a925c58932875fef9/src/bin/default.js#L33
which is resolved to
`js:C:\Users\boda2\WebstormProjects\marketplace-admin\node_modules\chimp\dist\lib\babel-register.js`
This value later comes to cucumber-js code:
https://github.com/cucumber/cucumber-js/blob/631f0aa77f16e14c86b4e79786f6085a681b6ce9/lib/cucumber/cli/configuration.js#L31
And here we have an issue: split by ":" results in "C" extracted instead of full path to file.

From my point of view real issue is in fragile parsing of cucumber-js project, but there is a fix on chimp side for my case by providing cli option compiler: 
`chimp --compiler=js:chimp/dist/lib/babel-register.js`

I'm not sure if this will work with chimp installed globally or you ever going to address this issue and change defaults accordingly to my workaround, so no pull request atm.

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
     "chokidar": "~1.0.6",
     "colors": "1.1.2",
     "commander": "^2.9.0",
-    "cucumber": "xolvio/cucumber-js#v0.9.4-chimp.1",
+    "cucumber": "xolvio/cucumber-js#v0.9.4-chimp.2",
     "exit": "^0.1.2",
     "fibers": "^1.0.7",
     "freeport": "~1.0.5",

